runtime/doc/ui.txt

```diff
@@ -38,6 +38,8 @@ with these (optional) keys:
 
 							*ui-ext-options*
 - `ext_cmdline`		Externalize the cmdline. |ui-cmdline|
+- `ext_elementtype`	Gives details about UI element types. |ui-elementtype|
```

Comment:
not clear why "types" is appended everywhere. It's emitted info about UI elements, right?

```suggestion
- `ext_elements`	Gives details about UI element. |ui-elementtype|
```

Reply:
In my programmer's mind, we get information about the type of the UI element, not the actual UI element itself, or even details about a given element. But I agree it's a bit redundant, so I can remove it.